src/components/scenes/ScanScene.js

```diff
@@ -48,10 +48,10 @@ export const LOGIN_QR = 'loginQR'
 
 export class Scan extends React.Component<Props> {
   componentDidUpdate(prevProps: Props) {
-    const { route } = this.props
+    const { navigation, route } = this.props
     const { data } = route.params
-    if (data !== prevProps.route.params.data && Actions.currentScene !== 'DrawerOpen') {
-      Actions.drawerClose()
+    if (data !== prevProps.route.params.data) {
```

Comment:
Why is it ok to remove the 'Actions.currentScene !== 'DrawerOpen'' without a replacement?

Reply:
Because `Actions.currentScene !== 'DrawerOpen'`  is always `true`.